.circleci/config.yml

```diff
@@ -236,6 +238,7 @@ jobs:
   Lint Rust with clippy:
     executor: docker
     steps:
+      - full-checkout
```

Comment:
We should checkout first, to make sure the `Cargo.lock` exists. We end up checksuming the `Cargo.lock` and use the checksum as a part of the key cache